invoices/invoiceregistry_test.go

```diff
@@ -663,14 +673,73 @@ func TestUnknownInvoice(t *testing.T) {
 	}
 }
 
+// TestKeySend tests receiving a spontaneous payment.
+func TestKeySend(t *testing.T) {
```

Comment:
Missing a case for rejecting incoming keysend payments when we have the flag off.

Reply:
Added